src/sentry_plugins/trello/plugin.py

```diff
@@ -14,6 +14,8 @@
 
 SETUP_URL = "https://github.com/getsentry/sentry/blob/master/src/sentry_plugins/trello/Trello_Instructions.md"  # NOQA
 
+LABLEX_REGEX = re.compile("\w+/https://trello.com/")
```

Comment:
Still missing the `r` prefix on the string, which may mess up with your `\w` part. You've also haven't escaped the `.` in `trello.com`, opening the door to matches like `trelloocom` etc.